matrix_sdk_base/src/client.rs

```diff
@@ -695,6 +695,25 @@ impl BaseClient {
         // }
     }
 
+    /// Handle a m.direc event, updating room states if necessary.
```

Comment:
```suggestion
    /// Handle a m.direc event, updating the room states if necessary.
```

Reply:
(This event is not tied to a room but multiple rooms, made rooms plural instead.)